button/generic-data-pipeline
# Migrating daily_orders and daily_transactions over from chartio repo.

Diff purely to bring daily_orders and daily_transactions jobs over from chartio repo.

Testing includes:
1. Bring up local mysql db. 
2. Source the proper environment variables.
3. Run db-migrate if you don't have the chartio db locally.
2. Run `python manage.py daily_transactions` and `python manage.py daily_orders` then confirm that both local tables reflect as expected.
